### PR TITLE
feat: add has_pass python helper

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: python/py_helpers.py [hunk 1]
```diff
@@ -99,6 +99,11 @@ def get_variable(self, name):
     def has_function(self, name):
         return self.find_function(name) != Node()
 
+    # Checks if the current scope contains a "pass" statement
+    
+    def has_pass(self):
+        return any(isinstance(node, ast.Pass) for node in self.tree.body)
```
Comment (ojeytonwilliams) [inline python/py_helpers.py:105]: This raises an exception if `self.tree.body` is missing, right? 

If that's correct, how about just returning false if there isn't a body?

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: python/py_helpers.py (hunk 1 above)
Comment (Dario-DC) [reply python/py_helpers.py:105]:
```suggestion
        try:
            return any(isinstance(node, ast.Pass) for node in self.tree.body)
        except AttributeError:
            return False
```
Is something like this acceptable? @ojeytonwilliams

Comment (Dario-DC) [reply python/py_helpers.py:105]:
```suggestion
        if getattr(self.tree, 'body', False):
            return any(isinstance(node, ast.Pass) for node in self.tree.body)
        return False
```
Probably better...

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**